### ISSUE TITLE
[BUG]: Like Button upvotes multiple time

### ISSUE THREAD

**Issue by aryansharma220** [HITK-TECH-Community/Community-Website#947]
**Describe the bug**
A user should be able to upvote an answer only once but right now, i am able to upvote an answer multiple times

**To Reproduce**
1. Go to 'Q&A
2. Click on Upvote button on any answer
3. See error

**Expected behavior**
One should be allowed to upvote only one time

**Screenshots**
![image](https://github.com/HITK-TECH-Community/Community-Website/assets/133234867/da066401-a315-48bf-8213-7afe53d7a995)

**Desktop (please complete the following information):**
 - OS: windows 10
 - Browser edge

**Have you read the [Code of Conduct](https://github.com/HITK-TECH-Community/Community-Website/blob/main/code_of_conduct.md)?**
yes

**Comment by Kajol-Kumari** [comment]
Duplicate of https://github.com/HITK-TECH-Community/Community-Website/issues/919
